gstcefsrc.cc

```diff
@@ -487,23 +587,52 @@ run_cef (GstCefSrc *src)
     gchar* current_dir = g_get_current_dir();
 
     gchar* old_base_path = base_path;
-    base_path = g_build_filename(current_dir, base_path, NULL);
+    base_path = g_build_filename(current_dir, base_path, nullptr);
 
     g_free(current_dir);
     g_free(old_base_path);
   }
 
-  gchar* browser_subprocess_path = g_build_filename(base_path, "gstcefsubprocess", NULL);
+#ifdef __APPLE__
+  gchar* browser_subprocess_path = g_build_filename(base_path, "gstcefsubprocess.app/Contents/MacOS/gstcefsubprocess", nullptr);
+#else
+  gchar* browser_subprocess_path = g_build_filename(base_path, "gstcefsubprocess", nullptr);
+#endif
   if (const gchar *custom_subprocess_path = g_getenv ("GST_CEF_SUBPROCESS_PATH")) {
     g_setenv ("CEF_SUBPROCESS_PATH", browser_subprocess_path, TRUE);
     g_free (browser_subprocess_path);
     browser_subprocess_path = g_strdup (custom_subprocess_path);
   }
 
+  GST_WARNING_OBJECT(src, "CEF subprocess: %s\n", browser_subprocess_path);
```

Comment:
Probably want `GST_DEBUG_OBJECT` at most here ?

Reply:
Makes sense. I had this as a warning as I traced what data was making its way to libcef.